media/css/firefox/feedback.less

```diff
@@ -0,0 +1,271 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+@font-face {
+    font-family: 'FA-Icons-Contact';
+    src: url('/media/fonts/icons-contact.eot?#iefix') format('embedded-opentype'),
+         url('/media/fonts/icons-contact.woff') format('woff'),
+         url('/media/fonts/icons-contact.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+}
+
+main {
+    padding: 0 20px;
+}
+
+.headline {
+    .font-size(48px);
+    text-align: center;
+}
+
+.tagline {
+    .font-size(24px);
+    text-align: center;
+}
+
+.mozilla-share-cta {
+    text-align: center;
+    margin: 0 auto;
+}
+
+.primary {
+    .clearfix;
+    .span-all;
+    margin-bottom: @baseLine;
+    padding: 20px;
```

Comment:
If we're using variables here it should probably be <code>padding: @baseLine @gridGutterWidth;</code> to keep vertical and horizontal spacing consistent with sandstone. They both happen to be 20px so the output is redundant, but the two variables might not always match.